vector/src/main/java/im/vector/app/features/call/VectorCallViewModel.kt

```diff
@@ -202,6 +208,22 @@ class VectorCallViewModel @AssistedInject constructor(
         }
     }
 
+    private fun WebRtcCall.extractCallInfo(): VectorCallViewState.CallInfo {
+        val assertedIdentity = this.remoteAssertedIdentity
+        val matrixItem = if (assertedIdentity != null) {
+            val userId = if (MatrixPatterns.isUserId(assertedIdentity.id)) {
+                assertedIdentity.id!!
+            } else {
+                // Need an id starting with @
+                "@${assertedIdentity.displayName}"
+            }
+            MatrixItem.UserItem(userId, assertedIdentity.displayName, assertedIdentity.avatarUrl)
```

Comment:
Create a `fun AssertedIdentity.toMatrixItem()` in the SDK, as per every other supported types?

Reply:
I can, but it was a bit weird, I just need that for AvatarRendering...

Reply:
That's one of the main point of MatrixItem, to be given to the avatar renderer :)